tree/ntuple/v7/src/RNTuple.cxx

```diff
@@ -103,6 +106,37 @@ std::string ROOT::Experimental::RNTupleReader::GetInfo(const ENTupleInfo what) {
    return "";
 }
 
+
+void ROOT::Experimental::RNTupleReader::Print(std::ostream &output, char frameSymbol, int width)
+{
+   if (width < 30) {
+      std::cout << "The width is too small! Should be at least 30.\n";
```

Comment:
This should be printed to `output`, too (and there should be a unit test for it)